### PR TITLE
Fixes a bug

### PR DESCRIPTION
**Pull request by ChangelingRain**
You can't spamclick a revenant to death with the null rod now.

### CODE REVIEW

File: code/game/gamemodes/miniantags/revenant/revenant.dm [hunk 1]
```diff
@@ -181,8 +182,6 @@
 		spawn(30)
 			inhibited = 0
 			update_action_buttons_icon()
-	else
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/revenant/revenant.dm]: I'm pretty sure phil did this. phiiiiiiiiil.

**Comment by phil235** [comment]
Your method will break attackby, just add a `user.changeNext_move(CLICK_CD_MELEE)` after the istype(nullrod)

**Comment by ChangelingRain** [comment]
How will it break attackby?

**Comment by phil235** [comment]
Actually you're right, it doesn't break attackby. I'm just sad this attackby is moving away from the standardization format I used in my refactor :cry: So I guess it's okay.

**Comment by optimumtact** [comment]
Congrats on your merge token and on breaking phils heart.

**Pull request merged**